src/base/bittorrent/sessionimpl.h

```diff
@@ -676,6 +688,9 @@ namespace BitTorrent
         CachedSettingValue<bool> m_blockPeersOnPrivilegedPorts;
         CachedSettingValue<bool> m_isAddTrackersEnabled;
         CachedSettingValue<QString> m_additionalTrackers;
+        CachedSettingValue<bool> m_isAddTrackersFromURLEnabled;
+        CachedSettingValue<QString> m_additionalTrackersURL;
+        CachedSettingValue<QString> m_additionalTrackersFromURL;
```

Comment:
It isn't a "setting" at all to be stored in a configuration file.

Reply:
I will change `m_additionalTrackersFromURL` to a normal member variable